config/locales/en.yml

```diff
@@ -538,6 +538,7 @@ en:
     timeline: Assignment Details
     timeline_enabled: Timeline enabled
     timeline_link: Timeline
+    timeline_tooltip: Timeline help you to keep track of your course by breaking the course duration into weeks which have proper agendas and tasks associated with them.
```

Comment:
"Enabling the Timeline will let you specify week-by-week activities and assign training modules. It is useful for Education Program courses and other long, highly structured programs."